### PR TITLE
Theme: SplitPane

### PR DESCRIPTION
**Pull request by bitpshr**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

This PR adds an initial theme for the SplitPane component.

screen shot 2017-07-21 at 3 07 39 pm

### CODE REVIEW

**Comment by bitpshr** [comment]
Bump @smhigley and @tomdye for a review. Thanks!

File: src/themes/dojo/splitPane.m.css [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
+	transition: border var(--transition-duration) var(--transition-easing);
+}
+
+.row > .divider {
+	border-left: 2.5px solid transparent;
+	border-right: 2.5px solid transparent;
+}
+
+.column > .divider {
+	border-bottom: 2.5px solid transparent;
```
Comment (tomdye) [inline src/themes/dojo/splitPane.m.css]: half pixels?

Comment (bitpshr) [inline src/themes/dojo/splitPane.m.css]: Fixed @tomdye.

**Comment by bitpshr** [comment]
Thanks for the review @tomdye. @smhigley and I spoke offline about an initial review as well.

**Pull request merged**